web/yo/app/scripts/services/mainutils.js

```diff
@@ -554,6 +581,13 @@ angular.module('oncokbApp')
             getHistoryData: getHistoryData,
             setUUIDInReview: setUUIDInReview,
             deleteUUID: deleteUUID,
-            updateMovingFlag: updateMovingFlag
+            updateMovingFlag: updateMovingFlag,
+            setAllUUIDs: setAllUUIDs,
```

Comment:
I don't suggest using `AllUUIDs`. This name does not give any information and later on would easily get confused with other concepts in the util.